taxonomies/food/ingredients.txt

```diff
@@ -27568,8 +27658,6 @@ yi: מעל
 zh: 麵粉
 carbon_footprint_fr_foodges_ingredient:fr: Farine de blé
 carbon_footprint_fr_foodges_value:fr: 1.2
-ciqual_proxy_food_code:en: 9410
-ciqual_proxy_food_name:fr: Farine de blé tendre ou froment T110
```

Comment:
Why did you remove this ?

Reply:
Because the generic "flour" is not the specific french flour type 110, and this causes all types of flour to inherit that specific type (when a more specific ciqual code is not defined)

Reply:
@McCio This is why the property is "ciqual_proxy_food_code" and not "ciqual_food_code": we know it's not the specific type, but it is a reasonable enough proxy that we can use for some applications. It's important to keep the proxies so that we can estimate nutrients etc.

Reply:
Thanks for explaining and for the patience, @stephanegigandet and @alexgarel, now I understand the logic behind it.
Fixed and rebased all over the updated main